cylc/flow/task_pool.py

```diff
@@ -1871,12 +1921,24 @@ def _get_task_proxy_db_outputs(
         self._load_historical_outputs(itask)
         return itask
 
-    def _standardise_prereqs(self, prereqs: 'List[str]') -> 'Set[PrereqTuple]':
-        """Convert trigger prerequisites to task messages."""
+    def _standardise_prereqs(self, prereqs: 'Iterable[str]') -> 'Set[Tokens]':
```

Comment:
Can be punted, but might be easier to include now?

I'm almost certain that at some point someone will want `--set=xtrigger/all` to be available.

Reply:
Yes, good point.